app-web/chromium/autobuild/patches/0014-AOSCOS-feat-introduce-miscellaneous-LoongArch-suppor.patch

```diff
@@ -233,43 +124,30 @@ index 7ef5eb238e..f2d1708e6c 100644
    }
  
    // Enum to pass as optional bool.
-diff --git a/components/variations/service/variations_field_trial_creator_base.cc b/components/variations/service/variations_field_trial_creator_base.cc
-index 4d71cf267b..84641e3378 100644
---- a/components/variations/service/variations_field_trial_creator_base.cc
-+++ b/components/variations/service/variations_field_trial_creator_base.cc
-@@ -114,6 +114,9 @@ Study::CpuArchitecture GetCurrentCpuArchitecture() {
-   if (process_arch == "ARM") {
-     return Study::ARM32;
+diff --git a/components/variations/service/variations_field_trial_creator.cc b/components/variations/service/variations_field_trial_creator.cc
+index 395171d59a..c2eef69c48 100644
+--- a/components/variations/service/variations_field_trial_creator.cc
++++ b/components/variations/service/variations_field_trial_creator.cc
+@@ -104,6 +104,9 @@ Study::CpuArchitecture GetCurrentCpuArchitecture() {
+   if (process_arch == "x86") {
+     return Study::X86_32;
    }
 +  if (process_arch == "LOONGARCH_64") {
 +    return Study::LOONGARCH64;
 +  }
-   if (process_arch == "x86") {
-     return Study::X86_32;
-   }
-diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
-index d930058bb0..a2a7c62c42 100644
---- a/content/common/user_agent.cc
-+++ b/content/common/user_agent.cc
-@@ -243,6 +243,8 @@ std::string GetCpuArchitecture() {
-               cpu_info.substr(2, 2) == "86") ||
-              base::StartsWith(cpu_info, "x86")) {
-     return "x86";
-+  } else if (base::StartsWith(cpu_info, "loong")) {
```

Comment:
This file is missing in Chromium 142 , we might need to check User-Agent used in LoongArch Chromium